beta/src/pages/learn/importing-and-exporting-components.md

```diff
@@ -4,24 +4,22 @@ title: Importing and Exporting Components
 
 <Intro>
 
-The magic of components lies in their reusability: you can create components that are composed of other components. But as you nest more and more components, it often makes sense to start splitting them into different files. This lets you keep your files easy to scan and reuse components in more places.
-
+הקסם של קומפוננטות טמון בשימוש החוזר שלהם: ניתן ליצור קומפנטטות המורכבות מקומפוננטות אחרות. אבל ככל שאתם מתחילים לקנן יותר ויותר קומפוננטות, לעתים קרובות הגיוני יותר להתחיל לפצל אותם לקבצים שונים. זה יאפשר לכם לשמור על הקבצים שלכם קריאים ולעשות שימוש חוזר בקומפוננטות במקומות נוספים.
 </Intro>
 
 <YouWillLearn>
 
-* What a root component file is
-* How to import and export a component
-* When to use default and named imports and exports
-* How to import and export multiple components from one file
-* How to split components into multiple files
+* מה היא קומפוננטת שורש
```

Comment:
instead of מה היא קומפוננטת שורש

מה היא קומפוננטת root

קומפוננטת שורש doesnt ring right